packages/datadog-plugin-aws-sdk/test/index.spec.js

```diff
@@ -311,6 +311,40 @@ describe('Plugin', () => {
             helpers.baseSpecs(semver, version, agent, getService, operation,
               serviceName, klass, sqsGetParams, key, metadata)
           })
+
+          describe('propagation', () => {
```

Comment:
Why a `describe` for a single test without description?

Reply:
No particular reason. I'll switch it to a single `it`.